Superlista/Models/BridgeModels/UserModelConverter.swift

```diff
@@ -26,7 +26,7 @@ class UserModelConverter {
         let customProducts: [ProductModel] = []
         let myLists: [ListModel] = []
         let sharedWithMe: [ListModel] = []
-                
+        
         id = user.id.recordName
         name = user.name ?? "arrumar esse inferno krl merda cu"
```

Comment:
HAHAHAHAHAHA FAVOR ARRUMAR ESSE INFERNO KRL MERDA CU ANTES DE SUBMETER PRA App Store

Sempre que tiver uma "", se for um nome de lista coloca "NovaLista" (ou o localizable equivalente), e quando for um nome de usuário, chamar a função getNickname()